src/api/config/initializers/flipper.rb

```diff
@@ -1,9 +1,4 @@
-Flipper.configure do |config|
-  config.default do
-    adapter = Flipper::Adapters::ActiveRecord.new
```

Comment:
@eduardoj @saraycp from the changelog and PR https://github.com/jnunemaker/flipper/pull/501 I understand that the adapter defaults to `Flipper::Adapters::Memory` if nothing is specified explicitly. But we are using the `ActiveRecord` one, or do I understand it wrong?

Reply:
We still require `Flipper::Adapters::ActiveRecord` with the line `gem 'flipper-active_record'` in the Gemfile. If we didn't have this `gem 'flipper-active_record'` line in the Gemfile, true, the adapter would use `Flipper::Adapters::Memory` as adapter.

Reply:
Adding the line `gem 'flipper-active_record'` to the `Gemfile` is also the documented way of adding an adapter. See https://github.com/jnunemaker/flipper#installation .

Reply:
Okay get it, you only need to do it if you want to further configure the adapter as also described here https://github.com/jnunemaker/flipper/blob/master/docs/active_record/README.md#usage

Reply:
Exactly.